opencga-app/app/scripts/azure/arm/README.md

```diff
@@ -60,6 +60,53 @@ Here are the properties that are defined for each t-shirt size:
 |             | node-size           | D2sv3        | D4sv3  | D4sv3  |
 |             | disk-type           | HDD          | HDD    | HDD    |
 
+Additionally you can deploy a custom size by specifying the `customTShirtSize` field and setting `deploymentSize=3`. The object has to contain all required fields. For an example see below. 

```

Comment:
Perhaps 0 should be custom, or even -1. As what if we need to add a size 3 of X-Large, and 4 of XXL etc...

Reply:
Yup makes sense I'll move custom to 0